test/integration/controllers/admin.spec.js

```diff
@@ -20,17 +28,177 @@ describe('Admin controller', () => {
 
   describe('admins endpoint', () => {
     it('should get admins', async () => {
```

Comment:
move admins to a variable, call it endpointUrl = '/admins/' and apply it for every 'describe', so tests follow the same schema everywhere

check user tests for the reference 

```
 describe(`GET ${endpointUrl}`, () => {
    it('should GET all admins', async () => {
```